test/util/range.cpp

```diff
@@ -213,6 +213,21 @@ void test_pointers()
     BOOST_CHECK(bgr::at(r2, 5) == 5);
 }
 
+void test_const_pointers()
```

Comment:
My proposal would be to merge the tests for pointers and const pointers as follows:

```
template <typename Iterator>
void test_pointers()
{
    typename std::iterator_traits<Iterator>::value_type arr[10]
        = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
    boost::iterator_range<Iterator> r1(arr, arr + 10);
    std::pair<Iterator, Iterator> r2(arr, arr + 10);

    BOOST_CHECK(bgr::front(r1) == 0);
    BOOST_CHECK(bgr::front(r2) == 0);
    BOOST_CHECK(bgr::back(r1) == 9);
    BOOST_CHECK(bgr::back(r2) == 9);
    BOOST_CHECK(bgr::at(r1, 5) == 5);
    BOOST_CHECK(bgr::at(r2, 5) == 5);
}
```
and then call the tests as:
```
   test_pointers<int*>();
   test_pointers<int const*>();
```
Also at the beginning of the file, `<iterator>` must be included.